packages/@lwc/perf-benchmarks/src/__benchmarks__/engine-dom/benchmark-wired-component/wireslist-create-10k.benchmark.js

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright (c) 2018, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: MIT
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/MIT
+ */
+
+import { createElement } from '@lwc/engine-dom';
+import WireAdaptersList from '@lwc/perf-benchmarks-components/dist/dom/benchmark/wireAdaptersList/wireAdaptersList.js';
+import Store from '@lwc/perf-benchmarks-components/dist/dom/benchmark/store/store.js';
+import { insertComponent, destroyComponent } from '../../../utils/utils.js';
+
+benchmark(`dom/wire-adapters-list/create/10k`, () => {
+    let wireAdaptersList;
+
+    before(() => {
+        wireAdaptersList = createElement('benchmark-wire-adapters-list', { is: WireAdaptersList });
+        return insertComponent(wireAdaptersList);
+    });
+
+    run(() => {
+        const store = new Store();
+        store.runLots();
```

Comment:
Ideally, `runLots` would be done in the `before` step. (I know other tests also do this wrong.) I'd be curious to know what % of the time in `run` is being spent in `runLots` versus the reactivity triggers for `wireAdaptersList.rows = store.data`. (BTW you can figure this out by running Tachometer in manual mode.)

Reply:
Generally, `runLots` consistently takes about `2.5-4.5ms`, while the entire `run` step ranges from `145-185ms`, so about `1-3%` of the time. The rest of the `run` step is taken up by rehydration.

Sample run (`runLots` is the 2nd light green block directly to the left of `rehydrate`:
<img width="1233" alt="Screenshot 2023-05-09 at 1 30 33 PM" src="https://github.com/salesforce/lwc/assets/40873183/47407f76-2fe5-4121-b82b-d77b4cf2db83">

Reply:
Beautiful, I was just curious. In any case, it looks like object-vs-string has a minimal impact on rendering perf. 👍 

Thank you for satisfying my curiosity!